sat/cli/status/status_module.py

```diff
@@ -553,12 +552,22 @@ def rows(self):
                     try:
                         component_session = bos_client.get_session(session_id)
                     except APIError as err:
-                        LOGGER.warning('Could not retrieve BOS session for component %s: %s',
-                                       component_xname, err)
+                        LOGGER.debug('Could not retrieve BOS session for component %s: %s',
+                                     component_xname, err)
```

Comment:
As we discussed, since a single session can be associated with multiple components, we don't really want to log this as being specific to a single component. It is only logged for the first time a non-existing session is encountered. Let's change this to something like:

```suggestion
                        LOGGER.debug('Could not retrieve BOS session %s: %s',
                                     session_id, err)
```